### ISSUE TITLE
Last version update of the database did not pass

### ISSUE THREAD

**Issue by kostyanius** [metabase/metabase#36618]
### Describe the bug

We are using script to search for the last version of the metabase.jar, download it and restart the metabase service.  all the time it worked fine and updates passed ok, but the last update cause to 502 error, metabase just can not start and repeats the boot loop. 
<img width="516" alt="2023-12-09_14h22_24" src="https://github.com/metabase/metabase/assets/9063908/400079d3-3821-4216-ad9d-b4a111e444dc">

### To Reproduce

just download the last metabse.jar version and restart the app with existing metabase.db.mv.db

### Expected behavior

it should work as before

### Logs

logs are absent. metabase does not provide any logs except those that are appearing after service restart:

```
Dec 09 13:00:40  systemd[1]: Started Metabase server.
Dec 09 13:00:48  metabase[49662]: 2023-12-09 13:00:48,541 INFO metabase.util :: Maximum memory available to JVM: 482.0 MB
Dec 09 13:00:51  metabase[49662]: 2023-12-09 13:00:51,808 WARN db.env :: WARNING: Using Metabase with an H2 application database is not recommended for production deployments. For production deployments, we highly recommend using Postgres, MySQL, or MariaDB instead. If you decide to continue to use H2, please be sure to back up the database file regularly. For more information, see https://metabase.com/docs/latest/operations-guide/migrating-from-h2.html
Dec 09 13:00:52 metabase[49662]: 2023-12-09 13:00:52,102 INFO util.encryption :: Saved credentials encryption is DISABLED for this Metabase instance. 🔓
Dec 09 13:00:52 metabase[49662]:  For more information, see https://metabase.com/docs/latest/operations-guide/encrypting-database-details-at-rest.html
```

### Information about your Metabase installation

```JSON
v0.47.9/ - failed
v0.47.8/ - was working ok
OS: 
No LSB modules are available.
Distributor ID: Ubuntu
Description:    Ubuntu 22.04.3 LTS
Release:        22.04
Codename:       jammy

browser: does not matter at all, problem is reproducing everywhere. firefox, chrom, safari and so on.
```

### Severity

blocker!

### Additional context

du -sh metabase.jar (v0.47.9)
311M    metabase.jar
 du -sh metabase.jar.old (v0.47.8)
343M    metabase.jar.old

so, the difference in size between version of metabase.jar v0.47.9 and v0.47.8 is 32 megabytes! How is that possible? it should be bigger, not smaller!

**Comment by paoliniluis** [comment]
Increase your ram

**Comment by kostyanius** [comment]
switched intance type from t3.small (2 cpu, 2 gb ram) to t3.medium (2 cpu, 4 gb ram). but still facing the issue. so, lools like is not a ram issue. 

```

Dec 09 15:51:55 systemd[1]: Started Metabase server.
Dec 09 15:52:05 metabase[930]: 2023-12-09 15:52:05,322 INFO metabase.util :: Maximum memory available to JVM: 962.0 MB
Dec 09 15:52:08 metabase[930]: 2023-12-09 15:52:08,588 WARN db.env :: WARNING: Using Metabase with an H2 application database is not recommended for production deployments. For production deployments, we highly recommend using Postgres, MySQL, or MariaDB instead. If you decide to continue to use H2, please be sure to back up the database file regularly. For more information, see https://metabase.com/docs/latest/operations-guide/migrating-from-h2.html
Dec 09 15:52:08 metabase[930]: 2023-12-09 15:52:08,979 INFO util.encryption :: Saved credentials encryption is DISABLED for this Metabase instance. 🔓
Dec 09 15:52:08  For more information, see https://metabase.com/docs/latest/operations-guide/encrypting-database-details-at-rest.html
```

**Comment by kostyanius** [comment]
Here is the all error log: 

```
Dec 09 15:59:58  systemd[1]: Started Metabase server.

Dec 09 16:00:08  metabase[1761]: 2023-12-09 16:00:08,391 INFO metabase.util :: Maximum memory available to JVM: 962.0 MB
Dec 09 16:00:11  metabase[1761]: 2023-12-09 16:00:11,843 WARN db.env :: WARNING: Using Metabase with an H2 application database is not recommended for production deployments. For production deployments, we highly recommend using Postgres, MySQL, or MariaDB instead. If you decide to continue to use H2, please be sure to back up the database file regularly. For more information, see https://metabase.com/docs/latest/operations-guide/migrating-from-h2.html
Dec 09 16:00:12  metabase[1761]: 2023-12-09 16:00:12,144 INFO util.encryption :: Saved credentials encryption is DISABLED for this Metabase instance. 🔓
Dec 09 16:00:12  metabase[1761]:  For more information, see https://metabase.com/docs/latest/operations-guide/encrypting-database-details-at-rest.html
Dec 09 16:00:22  metabase[1761]: 2023-12-09 16:00:22,735 INFO driver.impl :: Registered abstract driver :sql  🚚
Dec 09 16:00:22  metabase[1761]: 2023-12-09 16:00:22,756 INFO driver.impl :: Registered abstract driver :sql-jdbc (parents: [:sql]) 🚚
Dec 09 16:00:22  metabase[1761]: 2023-12-09 16:00:22,766 INFO metabase.util :: Load driver :sql-jdbc took 159.9 ms
Dec 09 16:00:22  metabase[1761]: 2023-12-09 16:00:22,768 INFO driver.impl :: Registered driver :h2 (parents: [:sql-jdbc]) 🚚
Dec 09 16:00:22  metabase[1761]: 2023-12-09 16:00:22,821 INFO driver.impl :: Registered driver :mysql (parents: [:sql-jdbc]) 🚚
Dec 09 16:00:22  metabase[1761]: 2023-12-09 16:00:22,904 INFO driver.impl :: Registered driver :postgres (parents: [:sql-jdbc]) 🚚
Dec 09 16:00:26  metabase[1761]: 2023-12-09 16:00:26,518 INFO metabase.core ::
Dec 09 16:00:26  metabase[1761]: Metabase v0.47.9 (d05b06e ?)
Dec 09 16:00:26  metabase[1761]: Copyright © 2023 Metabase, Inc.
Dec 09 16:00:26  metabase[1761]: Metabase Enterprise Edition extensions are NOT PRESENT.
Dec 09 16:00:26  metabase[1761]: 2023-12-09 16:00:26,554 INFO metabase.core :: Starting Metabase in STANDALONE mode
Dec 09 16:00:26  metabase[1761]: 2023-12-09 16:00:26,652 INFO metabase.server :: Launching Embedded Jetty Webserver with config:
Dec 09 16:00:26  metabase[1761]:  {:port 3000}
Dec 09 16:00:26  metabase[1761]: 2023-12-09 16:00:26,845 INFO metabase.core :: Starting Metabase version v0.47.9 (d05b06e ?) ...
Dec 09 16:00:26  metabase[1761]: 2023-12-09 16:00:26,933 INFO metabase.core :: System info:
Dec 09 16:00:26  metabase[1761]:  {"file.encoding" "UTF-8",
Dec 09 16:00:26  metabase[1761]:  "java.runtime.name" "OpenJDK Runtime Environment",
Dec 09 16:00:26  metabase[1761]:  "java.runtime.version" "11.0.21+9-post-Ubuntu-0ubuntu122.04",
Dec 09 16:00:26  metabase[1761]:  "java.vendor" "Ubuntu",
Dec 09 16:00:26  metabase[1761]:  "java.vendor.url" "https://ubuntu.com/",
Dec 09 16:00:26  metabase[1761]:  "java.version" "11.0.21",
Dec 09 16:00:26  metabase[1761]:  "java.vm.name" "OpenJDK 64-Bit Server VM",
Dec 09 16:00:26  metabase[1761]:  "java.vm.version" "11.0.21+9-post-Ubuntu-0ubuntu122.04",
Dec 09 16:00:26  metabase[1761]:  "os.name" "Linux",
Dec 09 16:00:26  metabase[1761]:  "os.version" "6.2.0-1017-aws",
Dec 09 16:00:26  metabase[1761]:  "user.language" "en",
Dec 09 16:00:26  metabase[1761]:  "user.timezone" "Etc/UTC"}
Dec 09 16:00:26  metabase[1761]: 2023-12-09 16:00:26,936 INFO metabase.plugins :: Loading plugins in /opt/metabase/plugins...
Dec 09 16:00:27  metabase[1761]: 2023-12-09 16:00:27,791 ERROR middleware.log :: GET /api/health 503 1.3 ms (0 DB calls)
Dec 09 16:00:27  metabase[1761]: {:status "initializing", :progress 0.2}
Dec 09 16:00:27  metabase[1761]:
Dec 09 16:00:27  metabase[1761]: 2023-12-09 16:00:27,988 DEBUG plugins.lazy-loaded-driver :: Registering lazy loading driver :hive-like...
Dec 09 16:00:27  metabase[1761]: 2023-12-09 16:00:27,989 INFO driver.impl :: Registered abstract driver :hive-like (parents: [:sql-jdbc]) 🚚
Dec 09 16:00:27  metabase[1761]: 2023-12-09 16:00:27,990 DEBUG plugins.lazy-loaded-driver :: Registering lazy loading driver :sparksql...
Dec 09 16:00:27  metabase[1761]: 2023-12-09 16:00:27,991 INFO driver.impl :: Registered driver :sparksql (parents: [:hive-like]) 🚚
Dec 09 16:00:28  metabase[1761]: 2023-12-09 16:00:28,022 DEBUG plugins.lazy-loaded-driver :: Registering lazy loading driver :bigquery-cloud-sdk...
Dec 09 16:00:28  metabase[1761]: 2023-12-09 16:00:28,023 INFO driver.impl :: Registered driver :bigquery-cloud-sdk (parents: [:sql]) 🚚
Dec 09 16:00:28  metabase[1761]: 2023-12-09 16:00:28,033 DEBUG plugins.lazy-loaded-driver :: Registering lazy loading driver :sqlserver...
Dec 09 16:00:28  metabase[1761]: 2023-12-09 16:00:28,034 INFO driver.impl :: Registered driver :sqlserver (parents: [:sql-jdbc]) 🚚
Dec 09 16:00:28  metabase[1761]: 2023-12-09 16:00:28,062 DEBUG plugins.lazy-loaded-driver :: Registering lazy loading driver :mongo...
Dec 09 16:00:28  metabase[1761]: 2023-12-09 16:00:28,068 INFO driver.impl :: Registered driver :mongo  🚚
Dec 09 16:00:28  metabase[1761]: 2023-12-09 16:00:28,078 DEBUG plugins.lazy-loaded-driver :: Registering lazy loading driver :redshift...
Dec 09 16:00:28  metabase[1761]: 2023-12-09 16:00:28,080 INFO driver.impl :: Registered driver :redshift (parents: [:postgres]) 🚚
Dec 09 16:00:28  metabase[1761]: 2023-12-09 16:00:28,086 DEBUG plugins.lazy-loaded-driver :: Registering lazy loading driver :druid...
Dec 09 16:00:28  metabase[1761]: 2023-12-09 16:00:28,088 INFO driver.impl :: Registered driver :druid  🚚
Dec 09 16:00:28  metabase[1761]: 2023-12-09 16:00:28,124 DEBUG plugins.lazy-loaded-driver :: Registering lazy loading driver :snowflake...
Dec 09 16:00:28  metabase[1761]: 2023-12-09 16:00:28,125 INFO driver.impl :: Registered driver :snowflake (parents: [:sql-jdbc]) 🚚
Dec 09 16:00:28  metabase[1761]: 2023-12-09 16:00:28,133 INFO plugins.dependencies :: Metabase cannot initialize plugin Metabase Oracle Driver due to required dependencies. Metabase requires the Oracle JDBC driver in order to connect to Oracle databases, but we can't ship it as part of Metabase due to licensing restrictions. See https://metabase.com/docs/latest/administration-guide/databases/oracle.html for more details.
Dec 09 16:00:28  metabase[1761]: 2023-12-09 16:00:28,137 INFO plugins.dependencies :: Metabase Oracle Driver dependency {:class oracle.jdbc.OracleDriver} satisfied? false
Dec 09 16:00:28  metabase[1761]: 2023-12-09 16:00:28,138 INFO plugins.dependencies :: Plugins with unsatisfied deps: ["Metabase Oracle Driver"]
Dec 09 16:00:28  metabase[1761]: 2023-12-09 16:00:28,156 DEBUG plugins.lazy-loaded-driver :: Registering lazy loading driver :athena...
Dec 09 16:00:28  metabase[1761]: 2023-12-09 16:00:28,157 INFO driver.impl :: Registered driver :athena (parents: [:sql-jdbc]) 🚚
Dec 09 16:00:28  metabase[1761]: 2023-12-09 16:00:28,182 DEBUG plugins.lazy-loaded-driver :: Registering lazy loading driver :presto-jdbc...
Dec 09 16:00:28  metabase[1761]: 2023-12-09 16:00:28,188 INFO driver.impl :: Registered driver :presto-jdbc (parents: [:sql-jdbc]) 🚚
Dec 09 16:00:28  metabase[1761]: 2023-12-09 16:00:28,192 DEBUG plugins.lazy-loaded-driver :: Registering lazy loading driver :sqlite...
Dec 09 16:00:28  metabase[1761]: 2023-12-09 16:00:28,194 INFO driver.impl :: Registered driver :sqlite (parents: [:sql-jdbc]) 🚚
Dec 09 16:00:28  metabase[1761]: 2023-12-09 16:00:28,201 INFO plugins.dependencies :: Metabase cannot initialize plugin Metabase Vertica Driver due to required dependencies. Metabase requires the Vertica JDBC driver in order to connect to Vertica databases, but we can't ship it as part of Metabase due to licensing restrictions. See https://metabase.com/docs/latest/administration-guide/databases/vertica.html for more details.
Dec 09 16:00:28  metabase[1761]: 2023-12-09 16:00:28,202 INFO plugins.dependencies :: Metabase Vertica Driver dependency {:class com.vertica.jdbc.Driver} satisfied? false
Dec 09 16:00:28  metabase[1761]: 2023-12-09 16:00:28,204 INFO plugins.dependencies :: Plugins with unsatisfied deps: ["Metabase Oracle Driver" "Metabase Vertica Driver"]
Dec 09 16:00:28  metabase[1761]: 2023-12-09 16:00:28,211 DEBUG plugins.lazy-loaded-driver :: Registering lazy loading driver :googleanalytics...
Dec 09 16:00:28  metabase[1761]: 2023-12-09 16:00:28,212 INFO driver.impl :: Registered driver :googleanalytics  🚚
Dec 09 16:00:28  metabase[1761]: 2023-12-09 16:00:28,215 INFO metabase.core :: Setting up and migrating Metabase DB. Please sit tight, this may take a minute...
Dec 09 16:00:28  metabase[1761]: 2023-12-09 16:00:28,222 INFO db.setup :: Verifying h2 Database Connection ...
Dec 09 16:00:28  metabase[1761]: 2023-12-09 16:00:28,940 ERROR middleware.log :: GET /api/health 503 375.5 µs (0 DB calls)
Dec 09 16:00:28  metabase[1761]: {:status "initializing", :progress 0.3}
Dec 09 16:00:28  metabase[1761]:
Dec 09 16:00:30  metabase[1761]: 2023-12-09 16:00:30,003 ERROR middleware.log :: GET /api/health 503 327.5 µs (0 DB calls)
Dec 09 16:00:30  metabase[1761]: {:status "initializing", :progress 0.3}
Dec 09 16:00:30  metabase[1761]:
Dec 09 16:00:31  metabase[1761]: 2023-12-09 16:00:31,067 ERROR middleware.log :: GET /api/health 503 360.6 µs (0 DB calls)
Dec 09 16:00:31  metabase[1761]: {:status "initializing", :progress 0.3}
Dec 09 16:00:31  metabase[1761]:
Dec 09 16:00:31  metabase[1761]: 2023-12-09 16:00:31,086 INFO db.setup :: Successfully verified H2 2.1.214 (2022-06-13) application database connection. ✅
Dec 09 16:00:31  metabase[1761]: 2023-12-09 16:00:31,089 INFO db.setup :: Running Database Migrations...
Dec 09 16:00:31  metabase[1761]: 2023-12-09 16:00:31,092 INFO db.setup :: Setting up Liquibase...
Dec 09 16:00:31  metabase[1761]: 2023-12-09 16:00:31,579 ERROR metabase.core :: Metabase Initialization FAILED
Dec 09 16:00:31  metabase[1761]: org.h2.jdbc.JdbcBatchUpdateException: The database is read only; SQL statement:
Dec 09 16:00:31  metabase[1761]: UPDATE DATABASECHANGELOG SET FILENAME = ? [90097-214]
Dec 09 16:00:31  metabase[1761]:         at org.h2.jdbc.JdbcPreparedStatement.executeBatch(JdbcPreparedStatement.java:1269)
Dec 09 16:00:31  metabase[1761]:         at com.mchange.v2.c3p0.impl.NewProxyPreparedStatement.executeBatch(NewProxyPreparedStatement.java:2544)
Dec 09 16:00:31  metabase[1761]:         at clojure.java.jdbc$execute_batch.invokeStatic(jdbc.clj:598)
Dec 09 16:00:31  metabase[1761]:         at clojure.java.jdbc$execute_batch.invoke(jdbc.clj:591)
Dec 09 16:00:31  metabase[1761]:         at clojure.java.jdbc$db_do_execute_prepared_statement$fn__21715.invoke(jdbc.clj:1057)
Dec 09 16:00:31  metabase[1761]:         at clojure.java.jdbc$db_transaction_STAR_.invokeStatic(jdbc.clj:807)
Dec 09 16:00:31  metabase[1761]:         at clojure.java.jdbc$db_transaction_STAR_.invoke(jdbc.clj:776)
Dec 09 16:00:31  metabase[1761]:         at clojure.java.jdbc$db_transaction_STAR_.invokeStatic(jdbc.clj:789)
Dec 09 16:00:31  metabase[1761]:         at clojure.java.jdbc$db_transaction_STAR_.invoke(jdbc.clj:776)
Dec 09 16:00:31  metabase[1761]:         at clojure.java.jdbc$db_do_execute_prepared_statement.invokeStatic(jdbc.clj:1056)
Dec 09 16:00:31  metabase[1761]:         at clojure.java.jdbc$db_do_execute_prepared_statement.invoke(jdbc.clj:1042)
Dec 09 16:00:31  metabase[1761]:         at clojure.java.jdbc$db_do_prepared.invokeStatic(jdbc.clj:1080)
Dec 09 16:00:31  metabase[1761]:         at clojure.java.jdbc$db_do_prepared.invoke(jdbc.clj:1060)
Dec 09 16:00:31  metabase[1761]:         at clojure.java.jdbc$execute_BANG_$execute_helper__21783.invoke(jdbc.clj:1464)
Dec 09 16:00:31  metabase[1761]:         at clojure.java.jdbc$execute_BANG_.invokeStatic(jdbc.clj:1466)
Dec 09 16:00:31  metabase[1761]:         at clojure.java.jdbc$execute_BANG_.invoke(jdbc.clj:1435)
Dec 09 16:00:31  metabase[1761]:         at clojure.java.jdbc$execute_BANG_.invokeStatic(jdbc.clj:1456)
Dec 09 16:00:31  metabase[1761]:         at clojure.java.jdbc$execute_BANG_.invoke(jdbc.clj:1435)
Dec 09 16:00:31  metabase[1761]:         at metabase.db.liquibase$fn__51847$consolidate_liquibase_changesets_BANG___51852$fn__51853.invoke(liquibase.clj:215)
Dec 09 16:00:31  metabase[1761]:         at metabase.db.liquibase$fn__51847$consolidate_liquibase_changesets_BANG___51852.invoke(liquibase.clj:199)
Dec 09 16:00:31  metabase[1761]:         at metabase.db.setup$fn__52148$migrate_BANG___52153$fn__52154$fn__52155.invoke(setup.clj:75)
Dec 09 16:00:31  metabase[1761]:         at metabase.db.liquibase$fn__51745$do_with_liquibase__51750$fn__51751.invoke(liquibase.clj:66)
Dec 09 16:00:31  metabase[1761]:         at metabase.db.liquibase$fn__51745$do_with_liquibase__51750.invoke(liquibase.clj:59)
Dec 09 16:00:31  metabase[1761]:         at metabase.db.setup$fn__52148$migrate_BANG___52153$fn__52154.invoke(setup.clj:73)
Dec 09 16:00:31  metabase[1761]:         at metabase.db.setup$fn__52148$migrate_BANG___52153.doInvoke(setup.clj:52)
Dec 09 16:00:31  metabase[1761]:         at clojure.lang.RestFn.invoke(RestFn.java:445)
Dec 09 16:00:31  metabase[1761]:         at metabase.db.setup$fn__52212$run_schema_migrations_BANG___52217$fn__52218.invoke(setup.clj:131)
Dec 09 16:00:31  metabase[1761]:         at metabase.db.setup$fn__52212$run_schema_migrations_BANG___52217.invoke(setup.clj:125)
Dec 09 16:00:31  metabase[1761]:         at metabase.db.setup$fn__52265$setup_db_BANG___52270$fn__52271$fn__52274$fn__52275.invoke(setup.clj:156)
Dec 09 16:00:31  metabase[1761]:         at metabase.util.jvm$do_with_us_locale.invokeStatic(jvm.clj:239)
Dec 09 16:00:31  metabase[1761]:         at metabase.util.jvm$do_with_us_locale.invoke(jvm.clj:225)
Dec 09 16:00:31  metabase[1761]:         at metabase.db.setup$fn__52265$setup_db_BANG___52270$fn__52271$fn__52274.invoke(setup.clj:152)
Dec 09 16:00:31  metabase[1761]:         at metabase.db.setup$fn__52265$setup_db_BANG___52270$fn__52271.invoke(setup.clj:151)
Dec 09 16:00:31  metabase[1761]:         at metabase.db.setup$fn__52265$setup_db_BANG___52270.invoke(setup.clj:145)
Dec 09 16:00:31  metabase[1761]:         at metabase.db$setup_db_BANG_$fn__52298.invoke(db.clj:69)
Dec 09 16:00:31  metabase[1761]:         at metabase.db$setup_db_BANG_.invokeStatic(db.clj:64)
Dec 09 16:00:31  metabase[1761]:         at metabase.db$setup_db_BANG_.invoke(db.clj:55)
Dec 09 16:00:31  metabase[1761]:         at metabase.core$init_BANG__STAR_.invokeStatic(core.clj:113)
Dec 09 16:00:31  metabase[1761]:         at metabase.core$init_BANG__STAR_.invoke(core.clj:99)
Dec 09 16:00:31  metabase[1761]:         at metabase.core$init_BANG_.invokeStatic(core.clj:159)
Dec 09 16:00:31  metabase[1761]:         at metabase.core$init_BANG_.invoke(core.clj:154)
Dec 09 16:00:31  metabase[1761]:         at metabase.core$start_normally.invokeStatic(core.clj:171)
Dec 09 16:00:31  metabase[1761]:         at metabase.core$start_normally.invoke(core.clj:165)
Dec 09 16:00:31  metabase[1761]:         at metabase.core$_main.invokeStatic(core.clj:204)
Dec 09 16:00:31  metabase[1761]:         at metabase.core$_main.doInvoke(core.clj:198)
Dec 09 16:00:31  metabase[1761]:         at clojure.lang.RestFn.invoke(RestFn.java:397)
Dec 09 16:00:31  metabase[1761]:         at clojure.lang.AFn.applyToHelper(AFn.java:152)
Dec 09 16:00:31  metabase[1761]:         at clojure.lang.RestFn.applyTo(RestFn.java:132)
Dec 09 16:00:31  metabase[1761]:         at clojure.lang.Var.applyTo(Var.java:705)
Dec 09 16:00:31  metabase[1761]:         at clojure.core$apply.invokeStatic(core.clj:667)
Dec 09 16:00:31  metabase[1761]:         at clojure.core$apply.invoke(core.clj:662)
Dec 09 16:00:31  metabase[1761]:         at metabase.bootstrap$_main.invokeStatic(bootstrap.clj:31)
Dec 09 16:00:31  metabase[1761]:         at metabase.bootstrap$_main.doInvoke(bootstrap.clj:28)
Dec 09 16:00:31  metabase[1761]:         at clojure.lang.RestFn.invoke(RestFn.java:397)
Dec 09 16:00:31  metabase[1761]:         at clojure.lang.AFn.applyToHelper(AFn.java:152)
Dec 09 16:00:31  metabase[1761]:         at clojure.lang.RestFn.applyTo(RestFn.java:132)
Dec 09 16:00:31  metabase[1761]:         at metabase.bootstrap.main(Unknown Source)
Dec 09 16:00:31  metabase[1761]: org.h2.jdbc.JdbcSQLNonTransientException: The database is read only; SQL statement:
Dec 09 16:00:31  metabase[1761]: UPDATE DATABASECHANGELOG SET FILENAME = ? [90097-214]
Dec 09 16:00:31  metabase[1761]:         at org.h2.message.DbException.getJdbcSQLException(DbException.java:554)
Dec 09 16:00:31  metabase[1761]:         at org.h2.message.DbException.getJdbcSQLException(DbException.java:477)
Dec 09 16:00:31  metabase[1761]:         at org.h2.message.DbException.get(DbException.java:223)
Dec 09 16:00:31  metabase[1761]:         at org.h2.message.DbException.get(DbException.java:199)
Dec 09 16:00:31  metabase[1761]:         at org.h2.message.DbException.get(DbException.java:188)
Dec 09 16:00:31  metabase[1761]:         at org.h2.engine.Database.checkWritingAllowed(Database.java:1724)
Dec 09 16:00:31  metabase[1761]:         at org.h2.table.Table.checkWritingAllowed(Table.java:1390)
Dec 09 16:00:31  metabase[1761]:         at org.h2.engine.User.hasTableRight(User.java:204)
Dec 09 16:00:31  metabase[1761]:         at org.h2.engine.User.checkTableRight(User.java:190)
Dec 09 16:00:31  metabase[1761]:         at org.h2.command.dml.Update.update(Update.java:60)
Dec 09 16:00:31  metabase[1761]:         at org.h2.command.dml.DataChangeStatement.update(DataChangeStatement.java:74)
Dec 09 16:00:31  metabase[1761]:         at org.h2.command.CommandContainer.update(CommandContainer.java:169)
Dec 09 16:00:31  metabase[1761]:         at org.h2.command.Command.executeUpdate(Command.java:252)
Dec 09 16:00:31  metabase[1761]:         at org.h2.jdbc.JdbcPreparedStatement.executeUpdateInternal(JdbcPreparedStatement.java:209)
Dec 09 16:00:31  metabase[1761]:         at org.h2.jdbc.JdbcPreparedStatement.executeBatchElement(JdbcPreparedStatement.java:1317)
Dec 09 16:00:31  metabase[1761]:         at org.h2.jdbc.JdbcPreparedStatement.executeBatch(JdbcPreparedStatement.java:1263)
Dec 09 16:00:31  metabase[1761]:         at com.mchange.v2.c3p0.impl.NewProxyPreparedStatement.executeBatch(NewProxyPreparedStatement.java:2544)
Dec 09 16:00:31  metabase[1761]:         at clojure.java.jdbc$execute_batch.invokeStatic(jdbc.clj:598)
Dec 09 16:00:31  metabase[1761]:         at clojure.java.jdbc$execute_batch.invoke(jdbc.clj:591)
Dec 09 16:00:31  metabase[1761]:         at clojure.java.jdbc$db_do_execute_prepared_statement$fn__21715.invoke(jdbc.clj:1057)
Dec 09 16:00:31  metabase[1761]:         at clojure.java.jdbc$db_transaction_STAR_.invokeStatic(jdbc.clj:807)
Dec 09 16:00:31  metabase[1761]:         at clojure.java.jdbc$db_transaction_STAR_.invoke(jdbc.clj:776)
Dec 09 16:00:31  metabase[1761]:         at clojure.java.jdbc$db_transaction_STAR_.invokeStatic(jdbc.clj:789)
Dec 09 16:00:31  metabase[1761]:         at clojure.java.jdbc$db_transaction_STAR_.invoke(jdbc.clj:776)
Dec 09 16:00:31  metabase[1761]:         at clojure.java.jdbc$db_do_execute_prepared_statement.invokeStatic(jdbc.clj:1056)
Dec 09 16:00:31  metabase[1761]:         at clojure.java.jdbc$db_do_execute_prepared_statement.invoke(jdbc.clj:1042)
Dec 09 16:00:31  metabase[1761]:         at clojure.java.jdbc$db_do_prepared.invokeStatic(jdbc.clj:1080)
Dec 09 16:00:31  metabase[1761]:         at clojure.java.jdbc$db_do_prepared.invoke(jdbc.clj:1060)
Dec 09 16:00:31  metabase[1761]:         at clojure.java.jdbc$execute_BANG_$execute_helper__21783.invoke(jdbc.clj:1464)
Dec 09 16:00:31  metabase[1761]:         at clojure.java.jdbc$execute_BANG_.invokeStatic(jdbc.clj:1466)
Dec 09 16:00:31  metabase[1761]:         at clojure.java.jdbc$execute_BANG_.invoke(jdbc.clj:1435)
Dec 09 16:00:31  metabase[1761]:         at clojure.java.jdbc$execute_BANG_.invokeStatic(jdbc.clj:1456)
Dec 09 16:00:31  metabase[1761]:         at clojure.java.jdbc$execute_BANG_.invoke(jdbc.clj:1435)
Dec 09 16:00:31  metabase[1761]:         at metabase.db.liquibase$fn__51847$consolidate_liquibase_changesets_BANG___51852$fn__51853.invoke(liquibase.clj:215)
Dec 09 16:00:31  metabase[1761]:         at metabase.db.liquibase$fn__51847$consolidate_liquibase_changesets_BANG___51852.invoke(liquibase.clj:199)
Dec 09 16:00:31  metabase[1761]:         at metabase.db.setup$fn__52148$migrate_BANG___52153$fn__52154$fn__52155.invoke(setup.clj:75)
Dec 09 16:00:31  metabase[1761]:         at metabase.db.liquibase$fn__51745$do_with_liquibase__51750$fn__51751.invoke(liquibase.clj:66)
Dec 09 16:00:31  metabase[1761]:         at metabase.db.liquibase$fn__51745$do_with_liquibase__51750.invoke(liquibase.clj:59)
Dec 09 16:00:31  metabase[1761]:         at metabase.db.setup$fn__52148$migrate_BANG___52153$fn__52154.invoke(setup.clj:73)
Dec 09 16:00:31  metabase[1761]:         at metabase.db.setup$fn__52148$migrate_BANG___52153.doInvoke(setup.clj:52)
Dec 09 16:00:31  metabase[1761]:         at clojure.lang.RestFn.invoke(RestFn.java:445)
Dec 09 16:00:31  metabase[1761]:         at metabase.db.setup$fn__52212$run_schema_migrations_BANG___52217$fn__52218.invoke(setup.clj:131)
Dec 09 16:00:31  metabase[1761]:         at metabase.db.setup$fn__52212$run_schema_migrations_BANG___52217.invoke(setup.clj:125)
Dec 09 16:00:31  metabase[1761]:         at metabase.db.setup$fn__52265$setup_db_BANG___52270$fn__52271$fn__52274$fn__52275.invoke(setup.clj:156)
Dec 09 16:00:31  metabase[1761]:         at metabase.util.jvm$do_with_us_locale.invokeStatic(jvm.clj:239)
Dec 09 16:00:31  metabase[1761]:         at metabase.util.jvm$do_with_us_locale.invoke(jvm.clj:225)
Dec 09 16:00:31  metabase[1761]:         at metabase.db.setup$fn__52265$setup_db_BANG___52270$fn__52271$fn__52274.invoke(setup.clj:152)
Dec 09 16:00:31  metabase[1761]:         at metabase.db.setup$fn__52265$setup_db_BANG___52270$fn__52271.invoke(setup.clj:151)
Dec 09 16:00:31  metabase[1761]:         at metabase.db.setup$fn__52265$setup_db_BANG___52270.invoke(setup.clj:145)
Dec 09 16:00:31  metabase[1761]:         at metabase.db$setup_db_BANG_$fn__52298.invoke(db.clj:69)
Dec 09 16:00:31  metabase[1761]:         at metabase.db$setup_db_BANG_.invokeStatic(db.clj:64)
Dec 09 16:00:31  metabase[1761]:         at metabase.db$setup_db_BANG_.invoke(db.clj:55)
Dec 09 16:00:31  metabase[1761]:         at metabase.core$init_BANG__STAR_.invokeStatic(core.clj:113)
Dec 09 16:00:31  metabase[1761]:         at metabase.core$init_BANG__STAR_.invoke(core.clj:99)
Dec 09 16:00:31  metabase[1761]:         at metabase.core$init_BANG_.invokeStatic(core.clj:159)
Dec 09 16:00:31  metabase[1761]:         at metabase.core$init_BANG_.invoke(core.clj:154)
Dec 09 16:00:31  metabase[1761]:         at metabase.core$start_normally.invokeStatic(core.clj:171)
Dec 09 16:00:31  metabase[1761]:         at metabase.core$start_normally.invoke(core.clj:165)
Dec 09 16:00:31  metabase[1761]:         at metabase.core$_main.invokeStatic(core.clj:204)
Dec 09 16:00:31  metabase[1761]:         at metabase.core$_main.doInvoke(core.clj:198)
Dec 09 16:00:31  metabase[1761]:         at clojure.lang.RestFn.invoke(RestFn.java:397)
Dec 09 16:00:31  metabase[1761]:         at clojure.lang.AFn.applyToHelper(AFn.java:152)
Dec 09 16:00:31  metabase[1761]:         at clojure.lang.RestFn.applyTo(RestFn.java:132)
Dec 09 16:00:31  metabase[1761]:         at clojure.lang.Var.applyTo(Var.java:705)
Dec 09 16:00:31  metabase[1761]:         at clojure.core$apply.invokeStatic(core.clj:667)
Dec 09 16:00:31  metabase[1761]:         at clojure.core$apply.invoke(core.clj:662)
Dec 09 16:00:31  metabase[1761]:         at metabase.bootstrap$_main.invokeStatic(bootstrap.clj:31)
Dec 09 16:00:31  metabase[1761]:         at metabase.bootstrap$_main.doInvoke(bootstrap.clj:28)
Dec 09 16:00:31  metabase[1761]:         at clojure.lang.RestFn.invoke(RestFn.java:397)
Dec 09 16:00:31  metabase[1761]:         at clojure.lang.AFn.applyToHelper(AFn.java:152)
Dec 09 16:00:31  metabase[1761]:         at clojure.lang.RestFn.applyTo(RestFn.java:132)
Dec 09 16:00:31  metabase[1761]:         at metabase.bootstrap.main(Unknown Source)
Dec 09 16:00:31  metabase[1761]: 2023-12-09 16:00:31,621 INFO metabase.core :: Metabase Shutting Down ...
Dec 09 16:00:31  metabase[1761]: 2023-12-09 16:00:31,621 INFO metabase.server :: Shutting Down Embedded Jetty Webserver
Dec 09 16:00:31  metabase[1761]: 2023-12-09 16:00:31,630 INFO metabase.core :: Metabase Shutdown COMPLETE
Dec 09 16:00:32  systemd[1]: metabase.service: Main process exited, code=exited, status=1/FAILURE
Dec 09 16:00:32  systemd[1]: metabase.service: Failed with result 'exit-code'.
Dec 09 16:00:32  systemd[1]: metabase.service: Consumed 1min 3.908s CPU time.
Dec 09 16:00:32  systemd[1]: metabase.service: Scheduled restart job, restart counter is at 15.
Dec 09 16:00:32  systemd[1]: Stopped Metabase server.
Dec 09 16:00:32  systemd[1]: metabase.service: Consumed 1min 3.908s CPU time.
Dec 09 16:00:32  systemd[1]: Started Metabase server.
```

**Comment by kostyanius** [comment]
i have increased the parametes for Java to ExecStart=/usr/bin/java -Xmx4096m -Xms2048m -jar /path/to/metabase.jar but still fails!
